collectors/property.yaml

```diff
@@ -0,0 +1,29 @@
+HostSystemPropertiesCollector:
```

Comment:
I really like the idea of having 3 types of different metrics!

Reply:
me too